test-result-summary-client/src/gitConf.json

```diff
@@ -0,0 +1,6 @@
+{
+	"Git":{
+		"user": "trssgit1",
+		"password": "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"
+	}
+}
```

Comment:
Please remove gitConf.json

Reply:
I am so sorry for including that file there. Removed it now. Thank you!

Reply:
No problem. Thanks @avishreekh for your quick response!